From becc61b31858199a1cff4278bb2239d05ff9d38f Mon Sep 17 00:00:00 2001
From: Snow <i@xkun.me>
Date: Sun, 15 May 2022 11:41:03 +0800
Subject: [PATCH 1/3] Define __USE_MISC in checkwidths.c

Define `__USE_MISC` manually in `checkwidths.c` for `ECHOCTL` and `ECHOKE`,
see `Kernel/API/POSIX/termios.h`.
---
 joe/util/checkwidths.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/joe/util/checkwidths.c b/joe/util/checkwidths.c
index 509447d..944faef 100644
--- a/joe/util/checkwidths.c
+++ b/joe/util/checkwidths.c
@@ -5,6 +5,8 @@
 
 */
 
+#define __USE_MISC	// for ECHOCTL and ECHOKE
+
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
-- 
2.36.1

